src/amber/router/context.cr

```diff
@@ -11,9 +11,11 @@ class HTTP::Server::Context
 
   property route : Radix::Result(Amber::Route)
   getter router : Amber::Router::Router
+  getter cookies : Amber::Router::Cookies::Store
 
   def initialize(@request : HTTP::Request, @response : HTTP::Server::Response)
     @router = Amber::Router::Router.instance
+    @cookies = Amber::Router::Cookies::Store.build(request, Amber::Server.key_generator)
```

Comment:
I don't like that this will initialize and probably decode cookies even when session, flash, or other middleware aren't being used. In an api for instance we won't want to have it initialize a cookies as it will slow us down and serve no purpose... Maybe allow this to be nil and let middleware set values if chosen?